addons/web/static/tests/core/colorlist.test.js

```diff
@@ -0,0 +1,83 @@
+/** @odoo-module */
+import { Component, xml } from "@odoo/owl";
+import { expect, test } from "@odoo/hoot";
+
+import { contains, mountWithCleanup } from "../web_test_helpers";
+import { ColorList } from "@web/core/colorlist/colorlist";
```

Comment:
first hoot (with on space after the odoo-module), then helpers, then space, then owl and colorlist